cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -81,227 +74,246 @@ CREATE CONSTRAINT FOR ... REQUIRE ...
-They can be dropped by using their name.
+
 
 a|
 label:functionality[]
-label:added[]
+label:deprecated[]
 [source, cypher, role="noheader"]
 ----
 CREATE CONSTRAINT
 ...
-OPTIONS "{" indexProvider: 'range-1.0' "}"
+OPTIONS "{" btree-option: btree-value[, ...] "}"
 ----
 a|
-<<indexes-future-indexes, FUTURE CONSTRAINT>>: Allows creating node key and uniqueness constraints backed by range indexes by providing the range index provider in the `OPTIONS` map.
-
+Node key and uniqueness constraints with b-tree options are deprecated and will be replaced in 5.0 by range options, see <<indexes-future-indexes, range indexes>>.
+In 4.4, the b-tree index-backed constraints are still the correct alternative to use.
 
-a|
-label:functionality[]
-label:added[]
+Will be replaced by:
 [source, cypher, role="noheader"]
 ----
-CREATE POINT INDEX ...
+CREATE CONSTRAINT
+...
+OPTIONS "{" range-option: range-value[, ...] "}"
 ----
-a|
-<<indexes-future-indexes, FUTURE INDEX>>: Allows creating point indexes on nodes or relationships with a particular label or relationship type, and property combination.
-They can be dropped by using their name.
+Constraints used for string properties will also require an additional text index to cover the string queries properly.
+Constraints used for point properties will also require an additional point index to cover the spatial queries properly, see <<indexes-future-indexes, point indexes>>.
 
 
 a|
 label:functionality[]
 label:deprecated[]
 [source, cypher, role="noheader"]
 ----
-CREATE BTREE INDEX ...
+distance(n.prop, point({x:0, y:0})
 ----
-.2+.^a|
-B-tree indexes are deprecated, partially replaced for now, and will be fully replaced in 5.0 by <<indexes-future-indexes, future indexes>>.
-In 4.4, b-tree indexes are still the correct alternative to use.
-
-B-tree indexes used for string queries are replaced by:
+a|
+Replaced by:
 [source, cypher, role="noheader"]
 ----
-CREATE TEXT INDEX ...
+point.distance(n.prop, point({x:0, y:0})
 ----
 
-B-tree indexes used for spatial queries will be replaced by:
+a|
+label:functionality[]
+label:deprecated[]
 [source, cypher, role="noheader"]
 ----
-CREATE POINT INDEX ...
+point({x:0, y:0}) <= point({x:1, y:1}) <= point({x:2, y:2})
 ----
-
-B-tree indexes used for general queries or property value types will be replaced by:
+a|
+Using inequality operators `<`, `<=`, `>`, and `>=` on spatial points is deprecated.
+Please instead use:
 [source, cypher, role="noheader"]
 ----
-CREATE RANGE INDEX ...
+point.withinBBox(point({x:1, y:1}), point({x:0, y:0}), point({x:2, y:2}))
 ----
 
-These new indexes may be combined for multiple use cases.
-
 a|
 label:functionality[]
 label:deprecated[]
+label:new[]
 [source, cypher, role="noheader"]
 ----
-CREATE INDEX
-...
-OPTIONS "{" btree-option: btree-value[, ...] "}"
+DROP CONSTRAINT
```

Comment:
I guess this was put last because it is both new and deprecated, but it feels slightly odd to not have it with the other constraint commands?